### ISSUE TITLE
Please let me keep lowercase tags and blog names

### ISSUE THREAD

**Issue by kseistrup** [Plume-org/Plume#721]
**Is your feature request related to a problem? Please describe.**

I am frustrated that Plume insists on capitalizing my blog names and tags, when I have taken care to enter lowercase words only. The funny thing is that the slug for blog entries are all lowercase, even when one or more words have been capitalized.

**Describe the solution you'd like**

I would like Plume to use the casing I have entered, instead of second-guessing me.

**Describe alternatives you've considered**

In my opinion, the entire URL ought to be lowercased, no matter what has been entered. The on-page text should be displayed exactly as entered by the author.

Cheers.

**Comment by AnaGelez** [comment]
Why not, I'm not sure why we changed the case of the tags originally. I just wonder how tags with spaces (for instance) will federate, but I guess we will see…

For blog IDs, we should probably suggest one by default based on the name, but allow to change it.

**Comment by kseistrup** [comment]
I'm fine with suggestions, as long as I get the final say.

**Comment by epsilon-phase** [comment]
Alright, it's easy enough to fix the tags, but for the blogs, where is it rejecting your case?

**Comment by kseistrup** [comment]
Any ETA?

**Comment by elegaanz** [comment]
Not really, sorry… we don't know when an issue will be solved, we just take them as we feel like resolving them. But this one shouldn't be too hard to fix, so it will probably be fixed sooner or later.

**Comment by kseistrup** [comment]
Thanks. I figured it would be something like that. But since this is such a small change, at least seen from a user's point of view, I was also somewhat disappointed that it wasn't included in the latest release. It's about the only thing that keeps me from using plume. But each has its time and season.

Happy bug squashing! 🐛

**Comment by MasterOfTheTiger** [comment]
Release early, release often. Whenever this is fix, it won't take such a long time to be released. (As far as I know).

**Comment by KitaitiMakoto** [comment]
In the latest release, tags' case is kept.

**Comment by kseistrup** [comment]
Much appreciated — thanks! :pray:

**Comment by KitaitiMakoto** [comment]
Ah, sorry, I closed this issue by mistake. Blog names are still modified.
